oat-sa/tao-core
# SQL export

__Related to task:__ https://oat-sa.atlassian.net/browse/NSA-116
__Description:__ SQL export for test results

__It depends on it:__ https://github.com/oat-sa/extension-tao-nec/pull/61

__How to test:__ 
1. create delivery and pass the test
2. open Export Table in Result tab
3. click 'Export SQL file'

__Expected Result:__ file with the SQL extension should be loaded. This should contain a SQL dump of the test results
